flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtil.java

```diff
@@ -169,6 +245,72 @@ public static void validateConsumerConfiguration(Properties config) {
 		}
 	}
 
+	/**
+	 * Validate the record publisher type.
+	 * @param config config properties
+	 * @return if `ConsumerConfigConstants.RECORD_PUBLISHER_TYPE` is set, return the parsed record publisher type. Else return polling record publisher type.
```

Comment:
@xiaolong-sn nit: Looks like you are using markdown in the javadoc. Usually you would write code using `{@code code goes here}` or links to classes/values `{@link MyClass}` like this

Reply:
I see, I'll use the {@code} here.